src/NzbDrone.Core/Notifications/Email/Email.cs

```diff
@@ -30,31 +36,144 @@ public override void OnReleaseImport(BookDownloadMessage message)
 
             var paths = Settings.AttachFiles ? message.BookFiles.SelectList(a => a.Path) : null;
 
-            _emailService.SendEmail(Settings, BOOK_DOWNLOADED_TITLE_BRANDED, body, false, paths);
+            SendEmail(Settings, BOOK_DOWNLOADED_TITLE_BRANDED, body, false, paths);
         }
 
         public override void OnHealthIssue(HealthCheck.HealthCheck message)
         {
-            _emailService.SendEmail(Settings, HEALTH_ISSUE_TITLE_BRANDED, message.Message);
+            SendEmail(Settings, HEALTH_ISSUE_TITLE_BRANDED, message.Message);
         }
 
         public override void OnDownloadFailure(DownloadFailedMessage message)
         {
-            _emailService.SendEmail(Settings, DOWNLOAD_FAILURE_TITLE_BRANDED, message.Message);
+            SendEmail(Settings, DOWNLOAD_FAILURE_TITLE_BRANDED, message.Message);
         }
 
         public override void OnImportFailure(BookDownloadMessage message)
         {
-            _emailService.SendEmail(Settings, IMPORT_FAILURE_TITLE_BRANDED, message.Message);
+            SendEmail(Settings, IMPORT_FAILURE_TITLE_BRANDED, message.Message);
         }
 
         public override ValidationResult Test()
         {
             var failures = new List<ValidationFailure>();
 
-            failures.AddIfNotNull(_emailService.Test(Settings));
+            failures.AddIfNotNull(Test(Settings));
 
             return new ValidationResult(failures);
         }
+
+        public ValidationFailure Test(EmailSettings settings)
+        {
+            const string body = "Success! You have properly configured your email notification settings";
+
+            try
+            {
+                SendEmail(settings, "Readarr - Test Notification", body);
+            }
+            catch (Exception ex)
+            {
+                _logger.Error(ex, "Unable to send test email");
+                return new ValidationFailure("Server", "Unable to send test email");
+            }
+
+            return null;
+        }
+
+        private void SendEmail(EmailSettings settings, string subject, string body, bool htmlBody = false, List<string> attachmentUrls = null)
+        {
+            var email = new MimeMessage();
+
+            email.From.Add(ParseAddress("From", settings.From));
+            email.To.AddRange(settings.To.Select(x => ParseAddress("To", x)));
+            email.Cc.AddRange(settings.CC.Select(x => ParseAddress("CC", x)));
+            email.Bcc.AddRange(settings.Bcc.Select(x => ParseAddress("BCC", x)));
+
+            email.Subject = subject;
+            email.Body = new TextPart(htmlBody ? "html" : "plain")
+            {
+                Text = body
+            };
+
+            if (attachmentUrls != null)
+            {
+                foreach (var url in attachmentUrls)
+                {
+                    email.Attachments.Add(new MimePart(url));
```

Comment:
specifically; figure this out

Reply:
http://www.mimekit.net/docs/html/P_MimeKit_MimeMessage_Attachments.htm
and
http://www.mimekit.net/docs/html/Creating-Messages.htm